paper/paper.md

```diff
@@ -0,0 +1,58 @@
+
+The Python ICLabel model is fully tested against and matches exactly the output produced in its MATLAB counterpart [@iclabel2019]. Moreover, this work builds the API on top of the robust MNE-Python ecosystem, enabling a seamless integration of automatic ICA analysis.
+
+# Statement of need
+
+Typically EEG and MEG data has many artifacts due to a variety of reasons, such as heart beats, eye blinks, line noise, channel noise, or muscle artifacts. A common signal processing technique for assisting in separating true brain signal from these types of signals is ICA [@Bell1995]. ICA performs blind-source separation decomposing the observed noisy electrophysiological signals into "sources", which are maximally independent. For example, this then allows researchers to decompose EEG signals into brain signals and a heart beat signal. Then by manually annotating a component time-series as a heart beat, one can remove the heart beat source from the signal and obtain their signals without the heart beat. This process typically is manually carried out and requires a human to label each component that comes out of ICA. This is subject to human error and is difficult to scale up when dealing with high-dimensional EEG, or MEG recordings.
+
+ICLabel was a proposed statistical model that uses neural networks and a crowdsourced training dataset to automatically label ICA components [@iclabel2019]. However, this package and functionality was previously only available in MATLAB, limiting its usage among Python neuroscience researchers. Moreover, the package relied on an outdated version of Tensorflow, which makes the model difficult to build on. In this work, we will build on top of the ICLabel model to improve its robustness when operating on different types of recording hardware and channel count. Moreover, we plan on building new types of models that improve its overall accuracy and performance with respect to auto-labeling neural and non-neural signals. The availability of a simple API for ICLabel will facilitate a strong benchmark to build future models and bring in new developers and contributors.
+
+MNE-Python is a general-purpose electrophysiology analysis package in Python that has a large core group of developers. Integration into MNE makes it likely that MNE-ICALabel will be maintained and continue to be improved. MNE-Python also has stability due to the funding it receives directly for development from institutions such as National Institutes of Health, the Chan Zuckerberg open-source initiative, the European Research Council and Agence Nationale de la Recherche in France.
+
+The developer team is excited to improve the state of the art in data handling and looking forward to welcoming new contributors and users.
+
+# Acknowledgements
+
+``MNE-ICALabel`` development is partly supported by
+the National Science Foundation under Grant # 2030859 to the Computing Research Association for the CIFellows Project.
```

Comment:
```suggestion
``MNE-ICALabel`` development is partly supported by
the National Science Foundation under Grant # 2030859 to the Computing Research Association for the CIFellows Project and by the Human Neuroscience Platform, Fondation Campus Biotech
Geneva, Geneva, Switzerland.
```

Do you mind adding an acknowledgment to the FCBG for opening a 6-month internship position on this project?